doc/developer-guides/hld/hld-APL_GVT-g.rst

```diff
@@ -118,7 +118,7 @@ API forwarding, or a split driver model, is another widely-used I/O
 virtualization technology. It has been used in commercial virtualization
 productions such as VMware*, PCoIP*, and Microsoft* RemoteFx*.
 It is a natural path when researchers study a new type of
-I/O virtualization usage—for example, when GPGPU computing in a VM was
+I/O virtualization usage. For example, when GPGPU computing in a VM was
```

Comment:
This now has an incomplete sentence starting with "For example". If you don't like the em dash, recommend putting the "for example..." in parentheses within the sentence.